source/fab/application.py

```diff
@@ -21,18 +26,42 @@ class Fab(object):
         '.f90': FortranAnalyser,
         '.F90': FortranPreProcessor,
     }
+    _compiler_map: Dict[str, Type[Command]] = {
+        '.f90': FortranCompiler,
+    }
+
+    def __init__(self,
+                 workspace: Path,
+                 target: str,
+                 exec_name: str,
+                 fpp_flags: str,
+                 fc_flags: str,
+                 ld_flags: str):
```

Comment:
As discussed with the developer, clearly passing the arguments for tools in this way is unsustainable. I have created a "Phase 3" milestone and attached a ticket regarding this configuration to it.